src/PriceCalculatorKata/Discount.cs

```diff
@@ -0,0 +1,22 @@
+namespace PriceCalculatorKata;
+public class Discount
+{
+    public static float UniversalDisscount { get; set; }
```

Comment:
I don't think this should be static, this can be changed though the app, maybe just pass as a parameter where it's needed, so you can pass different values?